gammapy/spectrum/spectrum_analysis.py

```diff
@@ -40,6 +46,33 @@ def main(args=None):
     run_spectral_fit_using_config(config)
 
 
+def filter_reflected_regions_analysis(analysis, n_min):
+    """Filter `~gammapy.spectrum.SpectrumAnalysis`
+
+    Only `~gammapy.spectrum.SpectrumObservation` with
+    n_reflected_regions >= n_min are kept
+
+    Parameters
+    ----------
+    analysis : `~gammapy.spectrum.SpectrumAnalysis`
+        Spectrum analysis with reflected regions background method
+    n_min : int
+        Minimum number of reflected regions
+
+    Returns
+    -------
+    mask : `np.array`
```

Comment:
I usually use `mask` for boolean mask arrays and `idx` or `index` for index arrays.
I think that's also the case in numpy / scipy / ....

So may I suggest you call this `idx` and call [numpy.nonzero](http://docs.scipy.org/doc/numpy-1.10.1/reference/generated/numpy.nonzero.html) here instead of [numpy.where](http://docs.scipy.org/doc/numpy-1.10.1/reference/generated/numpy.where.html)?